tests/hil/tests/rpc/README.md

```diff
@@ -1,3 +1,29 @@
 # RPC Test
 
 This tests the RPC service on the device.
+
+## Run Zephyr test locally
```

Comment:
What do you think about putting this at `tests/hil/platform/zephyr/README.md`? We already have build instructions there. We can link to that doc from here, but I think it will be easier to maintain if we have instructions centrally located.

Reply:
Sounds good. I've made the update and linked to that central README from the test readmes. I didn't link from the connection readme since that already has simplified syntax to run that test.